provider/GNUmakefile

```diff
@@ -56,9 +56,6 @@ $(OBJECTS) : | $(OBJ_PATH)
 
 CPPFLAGS+=$(INCLUDES)
 
-PRINT_BOOKENDS?=0
-CPPFLAGS+=-DPRINT_BOOKENDS=$(PRINT_BOOKENDS)
```

Comment:
you remove DPRINT_BOOKENDS, how can user show the debug info? is there other ways to enable this?

Reply:
Yes.
export CPPFLAGS="-DPRINT_BOOKENDS"
It should get documented!